SL/DO.pm

```diff
@@ -1142,6 +1142,7 @@ sub order_details {
     push @{ $form->{TEMPLATE_ARRAYS}{qty_nofmt} },       $form->{"qty_$i"};
     push @{ $form->{TEMPLATE_ARRAYS}{unit} },            $form->{"unit_$i"};
     push @{ $form->{TEMPLATE_ARRAYS}{partnotes} },       $form->{"partnotes_$i"};
+    push @{ $form->{TEMPLATE_ARRAYS}{partsgroup} },      $form->{"partsgroup_$i"};
```

Comment:
partsgroup muss auch noch in @arrays weiter oben (Zeile 1079). Sonst kann der Eintrag mit der Zwischenüberschrift beim Gruppieren von Positionen kaputt gehen (siehe ca. Zeile 1087 und 1107)